docs/app/pages/components/sections/tables/column-sorting/column-sorting.component.html

```diff
@@ -62,12 +60,25 @@
             <td>Defines the sort key for the column.</td>
         </tr>
 
+        <tr>
```

Comment:
Rename the 'sortKey' property to just 'key'. As part of the migration it's good to try simplifying attribute names, where they were previously more verbose than they needed to be.

Might also be good to add a bit more detail to the description to explain it's purpose as it currently isn't overly clear what it's purpose is.